### PR TITLE
[Feature]Add subcommand `--version`

### PR DESCRIPTION
**Pull request by chenk008**
## Why are these changes needed?

Add subcommand to print version, commit id, build time. It can help user to troubleshoot.

## Related issue number

Closes https://github.com/ray-project/kuberay/issues/137
## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [ ] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/Makefile [hunk 1]
```diff
@@ -58,7 +60,13 @@ test: manifests generate fmt vet ## Run tests.
 ##@ Build
 
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	go build                                    \
```
Comment (Jeffwan) [inline ray-operator/Makefile:63]: minor: Can we format `\` ?

**Review by Jeffwan — COMMENTED** [review]
Overall

File: ray-operator/Makefile (hunk 1 above)
Comment (Jeffwan) [reply ray-operator/Makefile:63]: image

Comment (chenk008) [reply ray-operator/Makefile:63]: Sorray, I have formatted it.

**Pull request merged**